### ISSUE TITLE
Add custom exceptions

### ISSUE THREAD

**Issue by kylehg** [pennlabs/penn-sdk-python#4]
Thereby allowing the user to more granularly handle API-specific exceptions

**Comment by adelq** [comment]
Not exactly, only laundry now has the custom exceptions, and they're not documented anywhere for clients to actually catch them. Reopening.
